cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -49,31 +49,28 @@ if node['cdo-apps']['daemon']
   if node.chef_environment == 'staging' && node.name == 'staging'
     cronjob at:'@reboot', do:home_dir('.dropbox-dist', 'dropboxd')
     cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'import_google_sheets')
-    cronjob at:'*/2 * * * *', do:deploy_dir('bin','run_server_generate_pdfs'), notify:'brendan@code.org'
-    cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','run_server_generate_curriculum_pdfs'), notify:'brian@code.org'
-    cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','collate_pdfs'), notify:'brian@code.org'
-    cronjob at:'*/2 * * * *', do:dashboard_dir('bin','build_scripts'), notify:'dev@code.org'
-    cronjob at:'*/5 * * * *', do:deploy_dir('bin','fetch-external-resources'), notify:'dev@code.org'
+    cronjob at:'*/2 * * * *', do:deploy_dir('bin','run_server_generate_pdfs'), notify:'dev+crontab@code.org'
+    cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','run_server_generate_curriculum_pdfs'), notify:'dev+crontab@code.org'
+    cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','collate_pdfs'), notify:'dev+crontab@code.org'
+    cronjob at:'*/2 * * * *', do:dashboard_dir('bin','build_scripts'), notify:'dev+crontab@code.org'
+    cronjob at:'*/5 * * * *', do:deploy_dir('bin','fetch-external-resources'), notify:'dev+crontab@code.org'
   end
 
   if node.chef_environment == 'production'
-    cronjob at:'*/15 * * * *', do:deploy_dir('bin','activity-monitor')
-    cronjob at:'@daily', do:deploy_dir('bin', 'send_workshop_reminder_emails')
-    cronjob at:'0 16 * * *', do:dashboard_dir('bin','scheduled_ops_emails')
+    cronjob at:'5 6 * * *', do:deploy_dir('bin', 'send_workshop_reminder_emails')
+    cronjob at:'15 16 * * *', do:dashboard_dir('bin','scheduled_ops_emails')
     cronjob at:'*/1 * * * *', do:deploy_dir('bin', 'index-users-in-solr')
-    cronjob at:'@daily', do:deploy_dir('bin', 'update-hoc-map')
+    cronjob at:'25 7 * * *', do:deploy_dir('bin', 'update-hoc-map')
```

Comment:
This runs only on daemon so shouldn't be necessary. I'll randomize the log file upload

> On Nov 30, 2015, at 19:37, philbogle <notifications@github.com> wrote:
> 
> In cookbooks/cdo-apps/templates/default/crontab.erb:
> 
> >      cronjob at:'*/1 * * * *', do:deploy_dir('bin', 'index-users-in-solr')
> > -    cronjob at:'@daily', do:deploy_dir('bin', 'update-hoc-map')
> > +    cronjob at:'25 7 * * *', do:deploy_dir('bin', 'update-hoc-map')
> Assuming this is someting that runs on multiple frontends, consider adding random jitter so that all of the frontends don't run at the same time; e.g. replace 25 with #{25 + rand(10)}. Consider doing that for other cronjobs where appropriate as well.
> 
> —
> Reply to this email directly or view it on GitHub.
>